analyzers/src/SonarAnalyzer.CFG/ShimLayer/OperationInterfaces.xml

```diff
@@ -3387,4 +3402,70 @@
       </Comments>
     </Property>
   </Node>
-</Tree>
+  <Node Name="ICollectionExpressionOperation" Base="IOperation"  HasType="true">
```

Comment:
The only non-formatting change in the file starts here.

Reply:
`?` were removed from the signatures on purpose